distro/pom.xml

```diff
@@ -57,6 +57,7 @@
         <communicationVersion>1.2.0-SNAPSHOT</communicationVersion>
         <terminologyServicesVersion>1.0.0-SNAPSHOT</terminologyServicesVersion>
         <cdssVersion>1.0.0-SNAPSHOT</cdssVersion>
+        <reportingCompatibilityVersion>2.10.0-SNAPSHOT</reportingCompatibilityVersion>
```

Comment:
@rahu1ramesh Do we have anything specific from the SNAPSHOT build? If not lets go with the released version. I think the last released version is 2.0.9 https://addons.openmrs.org/show/org.openmrs.module.reportingcompatibility

Reply:
Sure Mohan.